wom/src/test/scala/wdl4s/wdl/wom/WdlScatterWomSpec.scala

```diff
@@ -0,0 +1,209 @@
+package wdl4s.wdl.wom
+
+import lenthall.collections.EnhancedCollections._
+import cats.data.Validated.{Invalid, Valid}
+import org.scalatest.{FlatSpec, Matchers}
+import wdl4s.wdl.types.{WdlArrayType, WdlIntegerType, WdlStringType}
+import wdl4s.wdl.{WdlNamespace, WdlNamespaceWithWorkflow}
+import wdl4s.wom.graph.GraphNodePort.ScatterGathererPort
+import wdl4s.wom.graph._
+
+class WdlScatterWomSpec extends FlatSpec with Matchers {
+
+  behavior of "WdlNamespaces with scatters"
+
+  it should "convert gathered scatter outputs to workflow outputs" in {
+    val scatterTest =
+      """
+        |task foo {
+        |  Int i
+        |  command { ... }
+        |  output {
+        |    String out = i
+        |  }
+        |}
+        |
+        |workflow scatter_test {
+        |  Array[Int] xs
+        |  scatter (x in xs) {
+        |    call foo { input: i = x }
+        |  }
+        |}
+      """.stripMargin
```

Comment:
ToL: Just a reminder that with this IntelliJ default formatting, this last line doesn't strip anything, as the margin character "|" isn't provided.